lib/sprockets/cache.rb

```diff
@@ -51,7 +51,7 @@ class Cache
     # Internal: Cache key version for this class. Rarely should have to change
     # unless the cache format radically changes. Will be bump on major version
     # releases though.
-    VERSION = '4.0'
+    VERSION = '4.0.0'
```

Comment:
Unrelated?

Reply:
We need this because `concat_source_maps` expects a `x_sprockets_linecount` key in the map metadata. It wasn’t put there until this PR, so if a map gets pulled out of the old cache and attempts to get loaded into memory then it will fail.

By rev-ing the cache we can ensure the key will be there, since it’s added by `DefaultSourceMap`.